packages/salesforcedx-vscode-core/src/channels/channelService.ts

```diff
@@ -44,10 +44,14 @@ export class ChannelService {
     this.channel.appendLine(execution.command.toString());
     this.channel.appendLine('');
 
-    this.channel.appendLine(execution.command.toCommand());
+    this.channel.appendLine(
+      this.getExecutionTime() + ' ' + execution.command.toCommand()
+    );
 
     execution.processExitSubject.subscribe(data => {
-      this.channel.append(execution.command.toCommand());
+      this.channel.append(
```

Comment:
I looked at having this as part of the classes in `commandExecutor.ts`, having an internal variable (per class) to track start and end of execution and then leverage them whenever we wanted to print them. It turned out to be more complex and un-intuitive in case we wanted to use those timestamps somewhere else other than for the output view.
I then turned to the simplest approach which is adding it here on the right execution methods. Let me know if I should look at moving this somewhere else where it makes more sense.

cc @vazexqi